view/adminhtml/templates/info/adyen_moto.phtml

```diff
@@ -0,0 +1,107 @@
+<?php
+/**
+ *
+ * Adyen Payment module (https://www.adyen.com/)
+ *
+ * Copyright (c) 2022 Adyen N.V. (https://www.adyen.com/)
+ * See LICENSE.txt for license details.
+ *
+ * Author: Adyen <magento@adyen.com>
+ */
+
+// @codingStandardsIgnoreFile
+
+?>
+<?php
+/**
+ * @see \Magento\Payment\Block\Info
+ */
+
+?>
+<div>
+    <?= $block->escapeHtml($block->getMethod()->getTitle()); ?>
+</div>
+<?php
+$_info = $block->getInfo();
+$_isDemoMode = $block->isDemoMode();
+?>
+
+<?php if ($block->getMotoMerchantAccount() != ""): ?>
+    <div>
+        <?= $block->escapeHtml(__('Merchant Account: %1', $block->getMotoMerchantAccount())); ?><br/>
+    </div>
+<?php endif; ?>
+
+<?php if ($_pspReference = $_info->getAdditionalInformation('pspReference')): ?>
+    <div>
+        <?php if ($_isDemoMode): ?>
+            <?= $block->escapeHtml(__('Adyen PSP Reference: ')); ?>
+            <a href="https://ca-test.adyen.com/ca/ca/accounts/showTx.shtml?pspReference=<?= $block->escapeHtml(
```

Comment:
Can we use the `if` to only wrap the `<a>` tag? That way we don't have to duplicate the label.

Maybe even setting the URL in the header of this file with a ternary: `$_isDemoMode:https://ca-test.adyen.com/ca/ca/accounts/showTx.shtml?https://ca-live.adyen.com/ca/ca/accounts/showTx.shtml`;

I think this is done this way on an old template for CC but this could be improved here.